composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": "^5.6|^7.0",
-        "symfony/framework-bundle": "^2.8|^3.0|^4.0",
+        "symfony/framework-bundle": "^2.8|^3.0|^4.0|^5.1",
         "symfony/doctrine-bridge": "^2.8|^3.0|^4.0",
         "symfony/dependency-injection": "^2.8|^3.0|^4.0",
```

Comment:
We need to update to `^5.0` on all the `symfony/` libraries, including in require-dev (and I would use `^5.0`... I think it's not a big deal to still support that version).

Also, it's time to drop everything below 4.4. So I would use `"^4.4|^5.0"`

Reply:
Hi, yes sure. I will update this PR today. I just started it only.

Reply:
Updated. Thanks for review.